src/java.base/share/classes/java/lang/invoke/ProxyClassesDumper.java

```diff
@@ -120,6 +121,7 @@ public static String encodeForFilename(String className) {
                 }
             }
         }
+        sb.append(counter.incrementAndGet());
```

Comment:
```suggestion
        sb.append('$').append(counter.incrementAndGet());
```

Reply:
I suggested removing it in: https://github.com/openjdk/jdk/pull/12579#discussion_r1108941733

Reply:
Ah I missed that. Sure, I guess that's fine.